hnn_core/dipole.py

```diff
@@ -467,16 +467,11 @@ def savgol_filter(self, h_freq):
                                             self.sfreq)
         return self
 
-    def plot(self, tmin=None, tmax=None, layer='agg', decim=None, ax=None,
```

Comment:
this is technically a backwards incompatible change. If some user has `tmin` and `tmax` in their script, it will stop working now. The correct fix is to add a deprecation cycle. But if we want to assume that it's unlikely many users have this in their scripts, we should mark it as a "Bugfix" in `whats_new.rst` ...

Reply:
I thought one reason to have `tmin` was for burn in period @rythorpe may have an opinion ...

but it's true we can probably just set `plt.xlim((tmin, tmax))` to achieve the same

Reply:
See my comment in the previous PR [here](https://github.com/jonescompneurolab/hnn-core/pull/604#issuecomment-1462770587). I don't think we need `tmin` at the plotting level, however, it would be useful at the `simulate_dipole` level for a burn-in period. That's a very different feature though and should probably be implemented in a separate PR.

Reply:
I agree that we should do a deprecation cycle for these input arguments though.